src/desktop/modules/helper/permissionsHelper.ts

```diff
@@ -73,7 +70,31 @@ const PERMISSIONS_HELPER = {
       logger.error("Error while getting user", error);
       throw { message: `User read failed with ${error}`, status: error.code || 500 }
     }
-  }
+  },
+  // Fetch user data from server
+  async getManagedUsers(managedByUser: { access_token: string, userId: string }): Promise<ILoggedInUser> {
```

Comment:
In this method, the error messages should be relevant to the context which becomes hard to debug in the field